### PR TITLE
Tweed Scrolls Project

### PR DESCRIPTION
**Pull request by Ralph-Lee-UK**
Formatted text and facsimile files, added change note to head files.

### CODE REVIEW

**Comment by Ralph-Lee-UK** [comment]
@eu-genia sorry - checks have failed again - let me know what I need to do to put this right

**Comment by eu-genia** [comment]
When checks fail you can always see the report under Details (https://github.com/BetaMasaheft/Manuscripts/actions/runs/10595190493/job/29360403839?pr=2564) which shows which files are not valid and where the issue lies 

```
EMIP/magicscrolls/EMIPms00478/transkribusTextEMIPms00478.xml:213: parser error : Opening and ending tag mismatch: ab line 184 and div
         </div>
               ^
EMIP/magicscrolls/EMIPms00478/transkribusTextEMIPms00478.xml:399: parser error : Opening and ending tag mismatch: div line 4 and body
   </body>
          ^
EMIP/magicscrolls/EMIPms00478/transkribusTextEMIPms00478.xml:400: parser error : Opening and ending tag mismatch: body line 3 and text
</text>
       ^
EMIP/magicscrolls/EMIPms00478/transkribusTextEMIPms00478.xml:401: parser error : Premature end of data in tag text line 2

^
EMIP/magicscrolls/EMIPms00497/EMIPms00497.xml:80: element change: Relax-NG validity error : Did not expect element change there
EMIP/magicscrolls/EMIPms00497/EMIPms00497.xml:80: element change: Relax-NG validity error : Element teiHeader has extra content: change
EMIP/magicscrolls/EMIPms00497/EMIPms00497.xml fails to validate
```
so transkribusTextEMIPms00478.xml and EMIPms00497.xml have nesting issues (in EMIPms00497.xml the change element again seems to be in a wrong place)

**Comment by eu-genia** [comment]
I also note that you only replaced subtype to image for the first occurrence, but it should be for all occurences

File: EMIP/magicscrolls/EMIPms00469/transkribusTextEMIPms00469.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <text xmlns="http://www.tei-c.org/ns/1.0" xml:base="https://betamasaheft.eu/">
    <body>
       <div type="edition" xml:id="Transkribus" subtype="transkribus">
-         <div type="textpart" subtype="folio" n="1">
+         <div type="textpart" subtype="image" n="1">
```
Comment (eu-genia) [inline EMIP/magicscrolls/EMIPms00469/transkribusTextEMIPms00469.xml:5]: same for all the other folia

File: EMIP/magicscrolls/EMIPms00470/transkribusTextEMIPms00470.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <text xmlns="http://www.tei-c.org/ns/1.0" xml:base="https://betamasaheft.eu/">
    <body>
       <div type="edition" xml:id="Transkribus" subtype="transkribus">
-         <div type="textpart" subtype="folio" n="1">
+         <div type="textpart" subtype="image" n="1">
```
Comment (eu-genia) [inline EMIP/magicscrolls/EMIPms00470/transkribusTextEMIPms00470.xml:5]: other folia

File: EMIP/magicscrolls/EMIPms00471/transkribusTextEMIPms00471.xml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <text xmlns="http://www.tei-c.org/ns/1.0" xml:base="https://betamasaheft.eu/">
    <body>
       <div type="edition" xml:id="Transkribus" subtype="transkribus">
-         <div type="textpart" subtype="folio" n="1">
+         <div type="textpart" subtype="image" n="1">
```
Comment (eu-genia) [inline EMIP/magicscrolls/EMIPms00471/transkribusTextEMIPms00471.xml:5]: etc

File: EMIP/magicscrolls/EMIPms00472/transkribusTextEMIPms00472.xml [hunk 4]
```diff
@@ -2,7 +2,7 @@
 <text xmlns="http://www.tei-c.org/ns/1.0" xml:base="https://betamasaheft.eu/">
    <body>
       <div type="edition" xml:id="Transkribus" subtype="transkribus">
-         <div type="textpart" subtype="folio" n="1">
+         <div type="textpart" subtype="image" n="1">
```
Comment (eu-genia) [inline EMIP/magicscrolls/EMIPms00472/transkribusTextEMIPms00472.xml:5]: etc

File: EMIP/magicscrolls/EMIPms00473/transkribusTextEMIPms00473.xml [hunk 5]
```diff
@@ -2,7 +2,7 @@
 <text xmlns="http://www.tei-c.org/ns/1.0" xml:base="https://betamasaheft.eu/">
    <body>
       <div type="edition" xml:id="Transkribus" subtype="transkribus">
-         <div type="textpart" subtype="folio" n="1">
+         <div type="textpart" subtype="image" n="1">
```
Comment (eu-genia) [inline EMIP/magicscrolls/EMIPms00473/transkribusTextEMIPms00473.xml:5]: etc

File: EMIP/magicscrolls/EMIPms00474/transkribusTextEMIPms00474.xml [hunk 6]
```diff
@@ -2,7 +2,7 @@
 <text xmlns="http://www.tei-c.org/ns/1.0" xml:base="https://betamasaheft.eu/">
    <body>
       <div type="edition" xml:id="Transkribus" subtype="transkribus">
-         <div type="textpart" subtype="folio" n="1">
+         <div type="textpart" subtype="image" n="1">
```
Comment (eu-genia) [inline EMIP/magicscrolls/EMIPms00474/transkribusTextEMIPms00474.xml:5]: etc

File: EMIP/magicscrolls/EMIPms00477/transkribusTextEMIPms00477.xml [hunk 7]
```diff
@@ -2,7 +2,7 @@
 <text xmlns="http://www.tei-c.org/ns/1.0" xml:base="https://betamasaheft.eu/">
    <body>
       <div type="edition" xml:id="Transkribus" subtype="transkribus">
-         <div type="textpart" subtype="folio" n="1">
+         <div type="textpart" subtype="image" n="1">
```
Comment (eu-genia) [inline EMIP/magicscrolls/EMIPms00477/transkribusTextEMIPms00477.xml:5]: etc

File: EMIP/magicscrolls/EMIPms00478/transkribusTextEMIPms00478.xml [hunk 8]
```diff
@@ -2,144 +2,398 @@
...
+         <div type="textpart" subtype="image" n="1">
             <ab>
                <pb facs="#facs_1" n="1r"/>
-               <cb facs="#facs_1_r1" n="a"/>
-               <lb facs="#facs_1_line_1657183851763_13" n="1" break="no"/>በስመል አብ፡ ወወልድ፡ ወመን<lb facs="#facs_1_line_1657183918463_34" n="2" break="no"/>ፈስ፡ ቅዱስ፡ ፩አምላክ፡ ጸሎ</ab>
+               <cb facs="#facs_1_region_1659534443134_695" n="a"/>
+               <lb facs="#facs_1_line_1659534460965_698" n="1" break="no"/>ከፈይ፡ ወለተ፡ማርያም</ab>
          </div>
-         <div type="textpart" subtype="folio" n="2">
+         <div type="textpart" subtype="image" n="2">
```
Comment (eu-genia) [inline EMIP/magicscrolls/EMIPms00478/transkribusTextEMIPms00478.xml:11]: here all got replaced )

File: EMIP/magicscrolls/EMIPms00481/transkribusTextEMIPms00481.xml [hunk 9]
```diff
@@ -2,7 +2,7 @@
 <text xmlns="http://www.tei-c.org/ns/1.0" xml:base="https://betamasaheft.eu/">
    <body>
       <div type="edition" xml:id="Transkribus" subtype="transkribus">
-         <div type="textpart" subtype="folio" n="1">
+         <div type="textpart" subtype="image" n="1">
```
Comment (eu-genia) [inline EMIP/magicscrolls/EMIPms00481/transkribusTextEMIPms00481.xml:5]: etc

File: EMIP/magicscrolls/EMIPms00482/transkribusTextEMIPms00482.xml [hunk 10]
```diff
@@ -2,7 +2,7 @@
 <text xmlns="http://www.tei-c.org/ns/1.0" xml:base="https://betamasaheft.eu/">
    <body>
       <div type="edition" xml:id="Transkribus" subtype="transkribus">
-         <div type="textpart" subtype="folio" n="1">
+         <div type="textpart" subtype="image" n="1">
```
Comment (eu-genia) [inline EMIP/magicscrolls/EMIPms00482/transkribusTextEMIPms00482.xml:5]: etc

File: EMIP/magicscrolls/EMIPms00478/transkribusTextEMIPms00478.xml [hunk 11]
```diff
@@ -2,144 +2,398 @@
...
+				<lb facs="#facs_6_r1l13" n="10"/>ዘጌ፡ ዘጌ፡ 
+				<lb facs="#facs_6_r1l14" n="11"/>ዘጌ፡ ወኖ<lb facs="#facs_6_r1l15" n="12" break="no"/>ባማ፡ ንጉ<lb facs="#facs_6_r1l16" n="13" break="no"/>ሥከ፡ ወቢ<lb facs="#facs_6_r1l17" n="14" break="no"/>ካርያ፡ ወለ<lb facs="#facs_6_r1l18" n="15" break="no"/>ተ፡ ወርቅ፡ 
+				<lb facs="#facs_6_r1l19" n="16"/>ንግሥትከ፡ 
+				<lb facs="#facs_6_r1l20" n="17"/>ኢትክሰ፡ ወ<lb facs="#facs_6_r1l21" n="18" break="no"/>ኢትቈርጥ<lb facs="#facs_6_r1l22" n="19" break="no"/>ም፡ እደዊሃ፡ 
+				<lb facs="#facs_6_r1l23" n="20"/>ወእገሪሃ፡ 
+				<lb facs="#facs_6_r1l24" n="21"/>ወአስናኒሃ፡ 
+				<lb facs="#facs_6_r1l25" n="22"/>ዓዲ፡ ዘተር<lb facs="#facs_6_r1l26" n="23" break="no"/>ፈ፡ አባለ፡ 
+				<lb facs="#facs_6_r1l27" n="24"/>ሥጋሃ፡ ለ<lb facs="#facs_6_r1l28" n="25" break="no"/>አመተ፡ እ<lb facs="#facs_6_r1l29" n="26" break="no"/>ግዚአብሔ<lb facs="#facs_6_r1l30" n="27" break="no"/>ሮ፡ ወለተ፡ ማ<lb facs="#facs_6_line_1659535704318_1106" n="28" break="no"/>ርያም፡ ከፈይ<lb facs="#facs_6_line_1659535710950_1111" n="29" break="no"/>ጸሎት።
+			</ab>
+            <ab>
```
Comment (eu-genia) [inline EMIP/magicscrolls/EMIPms00478/transkribusTextEMIPms00478.xml:184]: check nesting

File: EMIP/magicscrolls/EMIPms00497/EMIPms00497.xml [hunk 12]
```diff
@@ -77,6 +77,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change when="2020-04-20" who="RL">Corrected ǝ character</change>
          <change when="2022-07-08" who="RL">Edited idno to include images</change>
          <change when="2022-09-27" who="RL">Added include code to add Transkribus transcription</change></revisionDesc>
+      <change when="2024-08-28" who="RL">formatted text and facsimile files</change>
```
Comment (eu-genia) [inline EMIP/magicscrolls/EMIPms00497/EMIPms00497.xml:80]: inside of revisionDesc

**Review by eu-genia — CHANGES_REQUESTED** [review]
FYI - for the future - I will have to do the update fully manually in this case as the app usually cannot manage with pull request containing more than 10-15 files at a time for the moment, the expansion process times out
No big problem just in case I am not around/online you will not be able to see the changes until I get around to pushing them

File: EMIP/magicscrolls/EMIPms00469/transkribusTextEMIPms00469.xml (hunk 1 above)
Comment (Ralph-Lee-UK) [reply EMIP/magicscrolls/EMIPms00469/transkribusTextEMIPms00469.xml:5]: I have corrected the style sheet now - I think these I did manually, and wasn't very careful!! Apologies

Comment (eu-genia) [reply EMIP/magicscrolls/EMIPms00469/transkribusTextEMIPms00469.xml:5]: Do you want to manually replace all occurences of subtype="folio" to subtype="image" in this and other files?

Comment (eu-genia) [reply EMIP/magicscrolls/EMIPms00469/transkribusTextEMIPms00469.xml:5]: see line 14, 25, etc

File: EMIP/magicscrolls/EMIPms00488/transkribusTextEMIPms00488.xml [hunk 13]
```diff
@@ -2,7 +2,7 @@
 <text xmlns="http://www.tei-c.org/ns/1.0" xml:base="https://betamasaheft.eu/">
    <body>
       <div type="edition" xml:id="Transkribus" subtype="transkribus">
-         <div type="textpart" subtype="folio" n="1">
+         <div type="textpart" subtype="image" n="1">
             <ab>
```
Comment (eu-genia) [inline EMIP/magicscrolls/EMIPms00488/transkribusTextEMIPms00488.xml:6]: see also line 22 etc

File: EMIP/magicscrolls/EMIPms00486/transkribusTextEMIPms00486.xml [hunk 14]
```diff
@@ -2,7 +2,7 @@
 <text xmlns="http://www.tei-c.org/ns/1.0" xml:base="https://betamasaheft.eu/">
    <body>
       <div type="edition" xml:id="Transkribus" subtype="transkribus">
-         <div type="textpart" subtype="folio" n="1">
+         <div type="textpart" subtype="image" n="1">
```
Comment (eu-genia) [inline EMIP/magicscrolls/EMIPms00486/transkribusTextEMIPms00486.xml:5]: other occurences

File: EMIP/magicscrolls/EMIPms00484/transkribusTextEMIPms00484.xml [hunk 15]
```diff
@@ -2,7 +2,7 @@
 <text xmlns="http://www.tei-c.org/ns/1.0" xml:base="https://betamasaheft.eu/">
    <body>
       <div type="edition" xml:id="Transkribus" subtype="transkribus">
-         <div type="textpart" subtype="folio" n="1">
+         <div type="textpart" subtype="image" n="1">
```
Comment (eu-genia) [inline EMIP/magicscrolls/EMIPms00484/transkribusTextEMIPms00484.xml:5]: other occurrences

File: EMIP/magicscrolls/EMIPms00469/transkribusTextEMIPms00469.xml (hunk 1 above)
Comment (Ralph-Lee-UK) [reply EMIP/magicscrolls/EMIPms00469/transkribusTextEMIPms00469.xml:5]: Yes, I have done that now

**Comment by Ralph-Lee-UK** [comment]
@eu-genia . I believe that I have made the required corrections.

File: EMIP/magicscrolls/EMIPms00469/transkribusTextEMIPms00469.xml (hunk 1 above)
Comment (eu-genia) [reply EMIP/magicscrolls/EMIPms00469/transkribusTextEMIPms00469.xml:5]: not for this file though

File: EMIP/magicscrolls/EMIPms00474/transkribusTextEMIPms00474.xml (hunk 6 above)
Comment (eu-genia) [reply EMIP/magicscrolls/EMIPms00474/transkribusTextEMIPms00474.xml:5]: here still folio

**Comment by Ralph-Lee-UK** [comment]
@eu-genia now it says that the validation has worked at least!

**Comment by eu-genia** [comment]
merged and synced

**Pull request merged**